monkey/monkey_island/cc/server_utils/island_logger.py

```diff
@@ -23,12 +24,22 @@ def setup_logging(data_dir: Path, log_level: str):
 
     formatter = _get_log_formatter()
 
-    log_file_path = data_dir / ISLAND_LOG_FILENAME
+    log_file_path = get_log_file_path(data_dir)
     _add_file_handler(logger, formatter, log_file_path)
 
     _add_console_handler(logger, formatter)
 
 
+def get_log_file_path(data_dir: Path) -> Path:
+    return data_dir / ISLAND_LOG_FILENAME
+
+
+def get_log_file_contents(log_file_path: Path) -> Mapping:
```

Comment:
```suggestion
def get_log_file_contents(log_file_path: Path) -> str:
```

Reply:
Do we even need this function? Could we just put this in the resource?

Reply:
https://github.com/guardicore/monkey/pull/2142#discussion_r934324401

Reply:
@shreyamalviya There's nothing specific to logging in this function. It doesn't get log file contents, it gets _any_ file contents. It should move to `monkey_island.cc.server_utils.file_utils` and be renamed.

Reply:
https://github.com/guardicore/monkey/pull/2151